rcode-reader/src/main/java/eu/rssw/pct/elements/MethodParameter.java

```diff
@@ -0,0 +1,104 @@
+package eu.rssw.pct.elements;
+
+import java.nio.ByteBuffer;
+import java.nio.ByteOrder;
+
+import eu.rssw.pct.DataType;
+import eu.rssw.pct.IParameter;
+import eu.rssw.pct.ParameterMode;
+import eu.rssw.pct.ParameterType;
+import eu.rssw.pct.RCodeInfo;
+
+public class MethodParameter extends AbstractElement implements IParameter {
+  private static final int PARAMETER_APPEND = 1;
+  private static final int PARAMETER_HANDLE = 2;
+  private static final int PARAMETER_BIND = 4;
+
+  private int extent;
+  // private int crc;
+  private int flags;
+  private int parameterType;
+  private int paramMode;
+  private int dataType;
+  // private int fullNameLength;
+  private int paramNum;
+
+  private String argumentName;
+
+  protected MethodParameter(byte[] segment, int currentPos, int textAreaOffset, ByteOrder order) {
+    this.parameterType = ByteBuffer.wrap(segment, currentPos, Short.BYTES).order(order).getShort();
+    this.paramMode = ByteBuffer.wrap(segment, currentPos + 2, Short.BYTES).order(order).getShort();
+    this.extent = ByteBuffer.wrap(segment, currentPos + 4, Short.BYTES).order(order).getShort();
+    this.dataType = ByteBuffer.wrap(segment, currentPos + 6, Short.BYTES).order(order).getShort();
+    // this.crc = ByteBuffer.wrap(segment, currentPos + 8, Short.BYTES).order(order).getShort();
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') This block of commented-out lines of code should be removed. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](http://sonar.riverside-software.fr/coding_rules#rule_key=squid%3ACommentedOutCodeLine)